admin/local/docker/compose/log/log-repl-controller.cnf

```diff
@@ -8,7 +8,7 @@ log4j.appender.CONSOLE.layout.ConversionPattern=[%d{yyyy-MM-ddTHH:mm:ss.SSSZ}] [
 # Tune log at the module level
 #log4j.logger.lsst.qserv.replica=DEBUG
 #log4j.logger.lsst.qserv.replica.DatabaseMySQL=DEBUG
-#log4j.logger.lsst.qserv.replica.MessengerConnector=DEBUG
+log4j.logger.lsst.qserv.replica.MessengerConnector=TRACE
```

Comment:
Do you always want TRACE logging, or was it for testing only?

Reply:
This particular config is used only by the integration test. I would like to keep the `TRACE` level on for now. This configuration won't "contaminate" any production deployments.